crd-generator/api/src/test/resources/k8svalidations.samples.fabric8.io-v1.yml

```diff
@@ -0,0 +1,161 @@
+#
```

Comment:
is this file used?

Reply:
Yes, see https://github.com/fabric8io/kubernetes-client/pull/5788#discussion_r1530262059